_config.yml

```diff
@@ -72,6 +72,10 @@ sponsors:
     link: //www.southampton.ac.uk/
     image: /images/sponsors/university-of-southampton.svg
 
+  - name: Adventurous Machines
```

Comment:
Question: What order should sponsors be in? Alphabetical? Order of tier? Aesthetic appeal of the logo?

Reply:
This issue came up at the last fundraising meeting and wasnt't fully resolved. Until it is I'd say put the logos in the date order that they have offered sponsorship?

Reply:
Ah ok. Yup that makes sense to me! :+1: